server/@types/express/index.d.ts

```diff
@@ -10,12 +10,14 @@ declare module 'express-session' {
 
 export declare global {
   namespace Express {
+    interface User {
```

Comment:
this overrides the Express.User which is used everywhere rather than just the user object attached to the request